clippy_lints/src/lib.deprecated.rs

```diff
@@ -0,0 +1,70 @@
+// This file was generated by `cargo dev update_lints`.
+// Use that command to update this file and do not edit by hand.
+// Manual edits will be overwritten.
+
+{
```

Comment:
Why is an extra scope necessary here?

Reply:
Yes. It's required because the include file must parse to a single expression.